about/licenses/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: LLNL Software Licensing
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+*This page contains information meant for LLNL Employees with instructions for
+how to mark their source code projects. It is not intended to serve as legal
+advice*
+
```

Comment:
Perhaps we can include some guidance about the status of these instructions? Are they a work in progress / what date are they active from? Can (must?) already released software change to meet these new standards?

Reply:
These will be active when they are published (PR is merged). I have gotten approval to make these the new standard.

There will be a transition period as we work to fix up existing projects. If anyone has cycles to work with projects cleaning this content up, please let me know!